Examples/_shared/GitHubAPI.swift

```diff
@@ -15,8 +15,14 @@ private func JSONResponseDataFormatter(_ data: Data) -> String {
     }
 }
 
-let gitHubProvider = MoyaProvider<GitHub>(plugins: [NetworkLoggerPlugin(configuration: .init(responseDataFormatter: JSONResponseDataFormatter,
-                                                                                             logOptions: .verbose))])
+let loggerPlugin: NetworkLoggerPlugin = {
```

Comment:
can we leave the inline initialization in the tests/examples? (plugins: [NetworkLoggerPlugin(configuration...]) - this will show us better what impact it will have on usage

Reply:
The inline declaration is back :)